recruiter/forms.py

```diff
@@ -27,3 +27,21 @@ def save(self, commit=True):
                           phone_number=self.cleaned_data['phone_number'])
         user2.save()
         return user2
+
+
+class CompanyRegistrationForm(forms.ModelForm):
```

Comment:
Why do we need so much code here? Why do we need to manually define the form fields for a ModelForm? Why do we need to override `save()`?

Reply:
you are right, that was the way I thought that should be done but It is easy and cleaner than the other way.

Reply:
Done.